### ISSUE TITLE
Multi-line text not wrapped in Email Notification.

### ISSUE THREAD

**Issue by watarutmnh** [verbb/formie#745]
**Description**
Multi-line text output by {{ contentHtml }} In Email Notification is not wrapped by `<p>` tag as single-line text was.
src/templates/_special/email-template/fields/multi-line-text.html

Wrapping by `<p>` tag would make the layout of Email content better.

**Comment by engram-design** [comment]
I'm not sure if the tags are missing from your issue? I'm not quite sure the issue here sorry

**Comment by watarutmnh** [comment]
Sorry! `<p>` tag that it is.

**Comment by engram-design** [comment]
Ah, of course. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.9](https://github.com/verbb/formie/releases/tag/1.5.9)
